sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/RequestOptions.java

```diff
@@ -16,27 +16,27 @@
      * The tracking ID sent with the request to help with debugging.
      */
     @JsonProperty(value = "")
-    private UUID xMsClientRequestId;
+    private UUID clientRequestId;
```

Comment:
Is the value set here actually setting `client-request-id` or `x-ms-client-request-id`?

Reply:
RequestOptions will match to internal RequestOptions, the name over there does not change. The JsonProperty here does not take affect.

Reply:
We just change the API exposed to customer